packages/web/server/api/edge-routers/numia-router.ts

```diff
@@ -0,0 +1,18 @@
+import { z } from "zod";
+
+import { createTRPCRouter, publicProcedure } from "~/server/api/trpc";
+import { getAverageStakingApr } from "~/server/queries/complex/numia/get-average-staking-apr";
+
+export const numiaRouter = createTRPCRouter({
+  getStakingApr: publicProcedure
```

Comment:
See comment above. Consider renaming to edge-routers/staking-router then this could be api.edge.staking.getApr.